.gitignore

```diff
@@ -31,3 +31,6 @@ cypress/videos
 
 # jetbrains
 .idea
+
+#yarn lock
+yarn.lock
```

Comment:
We actually push `yarn.lock` in the repository. Did you have previous experience where this shouldn't be part of the repo?